### PR TITLE
GH1173: Adds integration tests for Cake.Common.Tools.Cake.CakeAliases

### PR DESCRIPTION
**Pull request by devlead**
* releates to #1173

### CODE REVIEW

File: tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#load "./../../../utilities/xunit.cake"
+#load "./../../../utilities/paths.cake"
+
+Task("Cake.Common.Tools.Cake.CakeAliases.CakeExecuteScript")
+    .Does(() =>
+{
+    // Given
+    var path = Paths.Resources.Combine("./Cake.Common/Tools/Cake");
+    var file = path.CombineWithFilePath("./build.cake");
```
Comment (gep13) [inline tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake]: @devlead is this file missing from this PR?

File: tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    // When
+     CakeExecuteScript(file);
+});
+
+Task("Cake.Common.Tools.Cake.CakeAliases.CakeExecuteScript.Settings.Ok")
+    .Does(() =>
+{
+    // Given
+    var path = Paths.Resources.Combine("./Cake.Common/Tools/Cake");
+    var file = path.CombineWithFilePath("./test.cake");
```
Comment (gep13) [inline tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake]: @devlead is this file missing from this PR?

File: tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake (hunk 1 above)
Comment (devlead) [inline tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake]: Ah, Tools .gitignored will force add.

File: tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake (hunk 2 above)
Comment (devlead) [inline tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake]: Ah, Tools .gitignored will force add.

File: tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake (hunk 1 above)
Comment (devlead) [inline tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake]: @gep13 added, rebased & pushed.

File: tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake (hunk 2 above)
Comment (devlead) [inline tests/integration/Cake.Common/Tools/Cake/CakeAliases.cake]: @gep13 added, rebased & pushed.

**Comment by gep13** [comment]
@devlead LGTM!

**Pull request merged**